Fix acc test for datasource of network

keiichi-hikita opened the pull request:
In current implementation of network datasource test,
test is sometimes failed due to
existence of network which has same condition.
  (Like status is ACTIVE as well, tenant_id is same, like that)

So this pull request changes followings.

- remove status, tenant_id from argument of datasource
- remove relevant tests to the above changes
- remove them from argument section of corresponding document

rintooo commented:
LGTM. Confirmed acc tests passed.
To keep code clean, would you check below things, please?

1. Should we delete line 81-84, 91-92 in ecl/data_source_ecl_network_network_v2.go?
    ```
    - var status string
    - if v, ok := d.GetOk("status"); ok {
    -    status = v.(string)
    - }
    ```

    ```
    - Status:      status,
    - TenantID:    d.Get("tenant_id").(string),
    ```

2. It's better to add comment why this resource is necessary. Line 159 in ecl/data_source_ecl_network_network_v2.go.
    ```
    # write comment here
    resource "ecl_network_network_v2" "net2" {
    ```

keiichi-hikita commented:
Thanks for reviewing this.

> 1. Should we delete line 81-84, 91-92 in ecl/data_source_ecl_network_network_v2.go?
>    ```
>    - var status string
>    - if v, ok := d.GetOk("status"); ok {
>    -    status = v.(string)
>    - }
>    ```
>    ```
>    - Status:      status,
>    - TenantID:    d.Get("tenant_id").(string),
>    ```

You are absolutely right.
I removed needless logic from datasource file.

> 2. It's better to add comment why this resource is necessary. Line 159 in ecl/data_source_ecl_network_network_v2.go.
>    ```
>    # write comment here
>    resource "ecl_network_network_v2" "net2" {
>    ```

I think so too.
I added comment block above this test configuration function.
